cmd/openshift-tests/csi.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/openshift/origin/test/extended/csi"
+
+	"k8s.io/kubernetes/test/e2e/storage/external"
+)
+
+const (
+	manifestEnvVar       = "TEST_CSI_DRIVER_FILES"
+	installDriversEnvVar = "TEST_INSTALL_CSI_DRIVERS"
+)
+
+// Initialize openshift/csi suite, i.e. define CSI tests from TEST_CSI_DRIVER_FILES.
+func initCSITests() error {
```

Comment:
Who calls this?

Reply:
It's called from `initProvider`: https://github.com/openshift/origin/blob/d88cc07845b4e368145da435feb9e1bc318a54ac/cmd/openshift-tests/openshift-tests.go#L277-L286